tests/unit/directory_test.cc

```diff
@@ -74,13 +76,18 @@ int main(int ac, char** av) {
             });
         }
     };
+    fmt::print("--- Regular lister test ---\n");
```

Comment:
note to myself, i see the opportunity to transform this test into a `SEASTAR_THREAD_TEST_CASE` based tests.